packages/salesforcedx-utils-vscode/test/unit/commands/traceFlags.test.ts

```diff
@@ -20,7 +28,6 @@ describe('Trace Flags', () => {
   const USER_ID = 'abcd';
   let mockConnection: Connection;
   let sb: SinonSandbox;
-  let flags: TraceFlags;
```

Comment:
I removed flags (a TraceFlags) because it is what is being tested, and should be local to each test.